pkg/uprobetracer/ldcache_parser.go

```diff
@@ -0,0 +1,199 @@
+package uprobetracer
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"os"
+	"strings"
+	"unsafe"
+)
+
+const (
+	cache1Header    string = "ld.so-1.7.0"
```

Comment:
Where does this version `1.7.0` come from? Does it mean we only support a specific version of `ld.so`?

Reply:
It means `since 1.7.0`, and all the following versions are using the same header.